resources/openshift/templates/datavirt64-extensions-support-s2i.json

```diff
@@ -3,19 +3,23 @@
     "apiVersion": "v1",
     "metadata": {
         "annotations": {
-            "iconClass": "icon-jboss",
-            "description": "Application template for JBoss Data Virtualization 6.4 services built using S2I.  Includes support for installing extensions (e.g. third-party DB drivers) and the ability to configure certificates for serving secure content.",
-            "tags": "jdv,datavirt,jboss,xpaas",
-            "version": "1.4.0",
-            "openshift.io/display-name": "Red Hat JBoss Data Virtualization 6.4 (with SSL and Extensions)"
+            "iconClass": "icon-datavirt",
+            "tags": "jdv,datavirt,database,jboss",
+            "version": "1.4.8",
+            "openshift.io/display-name": "JBoss Data Virtualization 6.4 (with SSL and Extensions)",
+            "openshift.io/provider-display-name": "Red Hat, Inc.",
+            "description": "An example JBoss Data Virtualization application. For more information about using this template, see https://github.com/jboss-openshift/application-templates.",
+            "template.openshift.io/long-description": "This template defines resources needed to develop Red Hat Data Virtualization based application, including a build configuration, application deployment configuration, support for installing extensions (e.g. third-party DB drivers) and the ability to configure certificates for serving secure content.",
+            "template.openshift.io/documentation-url": "https://access.redhat.com/documentation/en/red-hat-jboss-data-virtualization/",
+            "template.openshift.io/support-url": "https://access.redhat.com"
         },
         "name": "datavirt64-extensions-support-s2i"
     },
     "labels": {
         "template": "datavirt64-extensions-support-s2i",
-        "xpaas": "1.4.0"
+        "xpaas": "1.4.8"
```

Comment:
I suspect the version is wrong, @rcernich  needs to specify what version should all Sprint 15 templates have.

Reply:
It probably makes the most sense to align this with the JDV release, so maybe remove xpaas and replace it with jdv, e.g. "jdv64": "1.0.0"